README.md

```diff
@@ -1,25 +1,35 @@
-Serlo editor as LTI tool Status: Early prototype
+Prototype: Serlo editor as LTI tool
 
 # Local dev setup
 
 Requirements:
 
 - Docker & Docker Compose
 
-1. Create file `.env` and copy content from `.env-template`
+1. Create local `.env` file and add required env values
```

Comment:
`.env-example` file would definitely be helpful. 
Or we could setup something like https://blog.1password.com/delete-your-example-env-file/ so new dev don't need to do anything here.

Reply:
Why did you delete [.env-template](https://github.com/serlo/serlo-editor-as-lti-tool/pull/87/files#diff-91d61e18014df9a046148581303d47288086fe8fa6bec55bb20aec54dc59e1f6)?

Reply:
Good suggestion! I had the same thing in mind. Let's talk about this with the other devs. :) 
In order to have a quick solution for now, I have local `.env` files for the different environments that I can share with you.

Reply:
Please do share